zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -4,26 +4,43 @@
 
 /obj/machinery/button/remote/elevator_panel/proc/despawn_passenger(var/mob/living/user)
 	var/mob/living/passenger = user
-	//var/mob/living/carbon/human/passengerplayer = user
-
-	for(var/obj/item/W in passenger)
-		if (istype(W, /obj/item/modular_computer/pda))
-			var/obj/item/modular_computer/pda/found_pda = W
-			found_pda.eject_id()
-		else if (istype(W, /obj/item/weapon/implant))
-			var/obj/item/weapon/implant/found_implant = W
-			found_implant.uninstall()
-			qdel(found_implant)
-		else if (istype(W, /obj/item/weapon/storage || /obj/item/clothing/suit/storage))
-			for(var/T in preserve_items)
-				for (var/obj/item/w_storage in W.contents)
-					if (istype(w_storage, T))
-						passenger.drop_from_inventory(w_storage)
+	var/obj/machinery/computer/cryopod/elevator_storage/lostFound
+
+	for(var/obj/machinery/computer/cryopod/elevator_storage/D in range(usr, 5))
```

Comment:
And what if you want to map the console somewhere further than 5 tiles away?

Reply:
It won't be 5 tiles away, it will be outside the elevator, as I showed in the pictures.

Reply:
It'd still allow for a lot more mapping flexibility if it iterated through the machines subsystem's machines list to find the console instead of having everything be within 5 tiles.

Reply:
Really not an issue... Again, if they want to map it that absurdly, they can do so and change the range